compiler/luci/pass/src/RemoveRedundantTranspose.test.cpp

```diff
@@ -106,6 +115,85 @@ void create_redundunt_transpose(loco::Graph *g, const std::vector<int32_t> &perm
   output->index(graph_output->index());
 }
 
+void create_redundunt_transpose_with_branch(loco::Graph *g, const std::vector<int32_t> &perm1,
```

Comment:
Comment painting(?) of graph would be help. Or is it same with the above graph?

Reply:
Oh, I will add this case more detail. Thanks for pointing out.